### PR TITLE
Add auth-ldap-user-dn-template in ldap_config.py

### PR DESCRIPTION
**Pull request by GnomeZworc**

### CODE REVIEW

File: configuration/ldap_config.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 AUTH_LDAP_BIND_DN = os.environ.get('AUTH_LDAP_BIND_DN', '')
 AUTH_LDAP_BIND_PASSWORD = os.environ.get('AUTH_LDAP_BIND_PASSWORD', '')
 
+# Set AUTH_LDAP_USER_DN_TEMPLATE
+AUTH_LDAP_USER_DN_TEMPLATE = os.environ.get('AUTH_LDAP_USER_DN_TEMPLATE', '')
```
Comment (thde) [inline configuration/ldap_config.py]: I think the default value here is `None`? https://django-auth-ldap.readthedocs.io/en/latest/reference.html#auth-ldap-user-dn-template

File: configuration/ldap_config.py [hunk 2]
```diff
@@ -15,6 +15,9 @@
 AUTH_LDAP_BIND_DN = os.environ.get('AUTH_LDAP_BIND_DN', '')
 AUTH_LDAP_BIND_PASSWORD = os.environ.get('AUTH_LDAP_BIND_PASSWORD', '')
 
+# Set AUTH_LDAP_USER_DN_TEMPLATE
```
Comment (thde) [inline configuration/ldap_config.py]: I'd use something like `Set a string template that describes any user’s distinguished name based on the username.` here.

**Comment by GnomeZworc** [comment]
@thde Can you recheck if it's ok now ?

**Comment by thde** [comment]
Looks good now, thanks!

**Pull request merged**